relationships.md

```diff
@@ -0,0 +1,280 @@
+        },
+        "rightType": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+        },
+        "self": {
+          "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/1"
+        }
+      },
+      "_embedded": {
+        "leftType": {
+          "id": 1,
+          "label": "Publication",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/1/relationshiptypes"
+            }
+          }
+        },
+        "rightType": {
+          "id": 2,
+          "label": "Person",
+          "type": "entitytype",
+          "_links": {
+            "self": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2"
+            },
+            "relationshiptypes": {
+              "href": "https://dspace7-entities.atmire.com/rest/api/core/entitytypes/2/relationshiptypes"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+The [relationship type](relationshiptypes.md) is embedded
+The 2 items are included as HAL links but are not embedded
+
+## Relationships per Relationship type
+**/api/core/relationships/search/byName?relationshipname=<:relationshipname>**
+
+A sample search would be https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationships/search/byName?relationshipname=isPersonOfOrgUnit
+The relationshipname parameter is mandatory
+
+It would respond with
+```json
+{
+  "_embedded": {
+    "relationships": [
+      {
+        "id": 590,
+        "leftId": "f2235aa6-6fe7-4174-a690-598b72dd8e44",
+        "relationshipTypeId": 0,
+        "rightId": "d30de96b-1e76-40ae-8ef9-ab426b6f9763",
+        "leftPlace": 1,
+        "rightPlace": 2,
+        "type": "relationship",
+        "_links": {
+          "relationshipType": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/relationshiptypes/5"
+          },
+          "self": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/relationships/590"
+          },
+          "leftItem": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/items/f2235aa6-6fe7-4174-a690-598b72dd8e44"
+          },
+          "rightItem": {
+            "href": "https://dspace7-entities.atmire.com/rest/api/core/items/d30de96b-1e76-40ae-8ef9-ab426b6f9763"
+          }
+        },
+        "_embedded": {
+          "relationshipType": {
+            "id": 5,
+            "leftLabel": "isOrgUnitOfPerson",
+            "rightLabel": "isPersonOfOrgUnit",
+            "leftMinCardinality": 0,
+            "leftMaxCardinality": 2147483647,
+            "rightMinCardinality": 0,
+            "rightMaxCardinality": 2147483647,
```

Comment:
These two max values should be null or simply left out.

Reply:
The cardinality values have been updated